packages/dataviews/src/filter-summary.js

```diff
@@ -74,9 +74,13 @@ function WithSeparators( { children } ) {
 
 export default function FilterSummary( { filter, view, onChangeView } ) {
 	const filterInView = view.filters.find( ( f ) => f.field === filter.field );
+	const otherFilters = view.filters.filter(
```

Comment:
Three refactorings (no behavioral changes) in this component:

- extracted `otherFilters`
- extracted `isActive`
- used `onChangeView({ ... })` instead of `onChangeView( ( currentView ) => { ... })`.